scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn-28.yaml

```diff
@@ -54,6 +54,69 @@ QFN-28-1EP_3x6mm_P0.5mm_EP1.7x4.75mm:
   #suffix: '_Pad{pad_x:.2f}x{pad_y:.2f}mm_HandSolder'
   #include_suffix_in_3dpath: 'False'
 
+QFN-28-1EP_4x4mm_P0.4mm_EP2.3x2.3mm:
+  device_type: 'QFN'
+  #manufacturer: 'man'
+  #part_number: 'mpn'
+  size_source: 'http://www.issi.com/WW/pdf/31FL3731.pdf#page=21'
+  ipc_class: 'qfn' # 'qfn_pull_back'
+  #ipc_density: 'least' #overwrite global value for this device.
+  # custom_name_format:
+  body_size_x:
+    nominal: 4
+    tolerance: 0
+  body_size_y:
+    nominal: 4
+    tolerance: 0
+  overall_height:
+    minimum: 0.7
+    nominal: 0.75
+    maximum: 0.8
+
+  lead_width:
+    minimum: 0.15
+    nominal: 0.2
+    maximum: 0.25
+  lead_len:
+    minimum: 0.3
+    nominal: 0.4
+    maximum: 0.5
+
+  EP_size_x:
+    minimum: 2.15
+    nominal: 2.3
+    maximum: 2.4
+  EP_size_x_overwrite: 2.3
```

Comment:
unnecessary. nominal is taken by default

Reply:
Sorry, just copied the one below. Done.